WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -267,6 +292,20 @@ public boolean onContextItemSelected(MenuItem item) {
         }
     }
 
+    private void setupSuggestionServiceAndAdapter() {
+        if (!isAdded()) return;
+
+        int remoteBlogId = -1;
+        String blogID = WordPress.getCurrentRemoteBlogId();
+        remoteBlogId = StringUtils.stringToInt(blogID, -1);
+
+        mSuggestionServiceConnectionManager = new SuggestionServiceConnectionManager(getActivity(), remoteBlogId);
+        mTagSuggestionAdapter = SuggestionUtils.setupTagSuggestions(remoteBlogId, getActivity(), mSuggestionServiceConnectionManager);
+        if (mTagSuggestionAdapter != null) {
```

Comment:
Since `mTagSuggestionAdapter` is only used within this method, can it also be downgraded from a class variable to a method variable?